### ISSUE TITLE
Rethinking Batch Normalization in Transformers

### ISSUE THREAD

**Issue by KeisukeShimokawa** [KeisukeShimokawa/pytorch-papers-challenge#50]
### 論文へのリンク

[[arXiv:2003.07845] Rethinking Batch Normalization in Transformers](https://arxiv.org/abs/2003.07845)

### 著者・所属機関

Sheng Shen, Zhewei Yao, Amir Gholami, Michael Mahoney, Kurt Keutzer

- University of California, Berkeley

### 投稿日時（YYYY-MM-DD）

2020-03-17

**Comment by KeisukeShimokawa** [comment]
[https://gist.github.com/AranKomat/3d5fbac5473aeb3303e92d13923424e3](https://gist.github.com/AranKomat/3d5fbac5473aeb3303e92d13923424e3)
